.gitignore

```diff
@@ -25,3 +25,20 @@ test-results
 
 /msvs/Debug/*
 /msvs/Release/*
+
+catboost/rust-package/target/
+
+catboost/rust-package/catboost-sys/target/
+
+catboost/app/catboost
+
+catboost/libs/model_interface/libcatboostmodel.dylib
```

Comment:
i suggest to use more generic `*.dylib*` instead of exact path to artifact.

Reply:
thanks will update